tutorials/install.md

```diff
@@ -154,9 +154,9 @@ Before [gz-rendering](https://github.com/gazebosim/gz-rendering) becomes availab
   cd build
   ```
 
-4. Before `gz-rendering` becomes available on conda-forge, we need to build it from source and specify the path containing `gz-rendering-config.cmake` in `CMAKE_PREFIX_PATH`, for cmake to find `gz-rendering`. That path could be `gz-rendering-install-path\lib\cmake\gz-rendering4`, for example.
+4. Before `gz-rendering` becomes available on conda-forge, we need to build it from source and specify the path containing `gz-rendering-config.cmake` in `CMAKE_PREFIX_PATH`, for cmake to find `gz-rendering`. That path could be `gz-rendering-install-path\lib\cmake\gz-rendering8`, for example.
```

Comment:
@mjcarroll the windows instructions are probably out of date? Is it possible to just do 
`conda install libgz-sensors<#>` / `conda install libgz-rendering<#>`?

Reply:
That is correct:

https://prefix.dev/channels/conda-forge/packages/gz-sensors7
https://prefix.dev/channels/conda-forge/packages/gz-rendering7

The harmonic versions will come when we have official tags, according to @traversaro

Reply:
Yes, conda-forge does not do prerelease in general, see https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds .

Reply:
updated windows instructions in [f975133](https://github.com/gazebosim/gz-sensors/pull/382/commits/f9751335bd34ae46768a3919dc52c3eabc1942fa)